plugin_test.go

```diff
@@ -97,6 +98,27 @@ var commits = []struct {
 		file: "README",
 		data: "Hello World!\n\nsomething is changed!\n",
 	},
+	// ### test lfs, please do not change order, otherwise TestCloneNonEmpty will fail ###
+	// checkout with lfs skip
+	{
+		path:     "test-assets/woodpecker-git-test-lfs",
+		clone:    "https://github.com/test-assets/woodpecker-git-test-lfs.git",
+		event:    "push",
+		commit:   "69d4dadb4c2899efb73c0095bb58a6454d133cef",
+		ref:      "refs/heads/main",
+		file:     "4M.bin",
+		dataSize: 132,
+	},
+	// checkout with lfs
```

Comment:
LFS does not seem to be tested here!

Reply:
hmm ba7d69980776138258b09c17ee5f909fd4ff3827 did missed it